model/project.go

```diff
@@ -535,10 +535,11 @@ func (c *YAMLCommandSet) UnmarshalYAML(unmarshal func(interface{}) error) error
 // TaskUnitDependency holds configuration information about a task/group that must finish before
 // the task/group that contains the dependency can run.
 type TaskUnitDependency struct {
-	Name          string `yaml:"name,omitempty" bson:"name"`
-	Variant       string `yaml:"variant,omitempty" bson:"variant,omitempty"`
-	Status        string `yaml:"status,omitempty" bson:"status,omitempty"`
-	PatchOptional bool   `yaml:"patch_optional,omitempty" bson:"patch_optional,omitempty"`
+	Name                           string `yaml:"name,omitempty" bson:"name"`
+	Variant                        string `yaml:"variant,omitempty" bson:"variant,omitempty"`
+	Status                         string `yaml:"status,omitempty" bson:"status,omitempty"`
+	PatchOptional                  bool   `yaml:"patch_optional,omitempty" bson:"patch_optional,omitempty"`
+	DependOnGeneratedTasksDisabled bool   `yaml:"depend_on_generated_tasks_disabled" bson:"depend_on_generated_tasks_disabled"`
```

Comment:
maybe just "include_generated_tasks" and we can have this be a bool pointer so the default is true? Or `omit_generated_tasks` so that false can still be the default

Reply:
Ok! `omit_generated_tasks` sounds elegant